<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nopayManager</class>
 <widget class="QWidget" name="nopayManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>860</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>应付款明细</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="child_widget_title" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>应付款明细</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>730</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="min_btn">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="max_restore_btn">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close_btn">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_9" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(54, 78, 111);</string>
     </property>
     <widget class="QDateEdit" name="nopay_enddateEdit">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>110</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>65</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>开始时间：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nopay_search_btn">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>80</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="nopay_add_btn">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>80</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="nopay_export_btn">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>80</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QDateEdit" name="nopay_startdateEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>110</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>65</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>结束时间：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nopay_del_btn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>80</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="nopay_tableView">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(54, 78, 111);</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
